test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -15,6 +15,7 @@ check_result_qemu() {
     umount $TESTDIR/root
     [[ -f $TESTDIR/failed ]] && cat $TESTDIR/failed
     ls -l $TESTDIR/journal/*/*.journal
+    journalctl -u testsuite --no-pager --file=$TESTDIR/journal/*/*.journal
```

Comment:
@evverx: Sure, moving the journal printing to e. g. `do_test()` sounds fine -- once that lands I'll take it out of the Debian test. But I wouldn't restrict it to one particular test (like here) and not to QEMU -- this applies to nspawn and other tests as well.